src/table/settings/gui_settings.ini

```
@@ -920,3 +920,62 @@ post_cb  = [](auto) { SetupWidgetDimensions(); ReInitAllWindows(true); }
 cat      = SC_BASIC
 startup  = true
 
+[SDTC_VAR]
+var      = gui.gamepad_deadzone
+type     = SLE_UINT8
+flags    = SettingFlag::NotInSave, SettingFlag::NoNetworkSync
+def      = 10
+min      = 0
+max      = 100
+interval = 5
+str      = STR_CONFIG_SETTING_GAMEPAD_DEADZONE
+strhelp  = STR_CONFIG_SETTING_GAMEPAD_DEADZONE_HELPTEXT
+strval   = STR_JUST_COMMA
```

This will give you the percentage with the symbol in the correct colour (orange to match the number, per other settings).

```suggestion
strval   = STR_CONFIG_SETTING_PERCENTAGE
```